tests/foreman/api/test_provisioningtemplate.py

```diff
@@ -634,6 +642,8 @@ def test_positive_template_check_rex_snippet(
             in rex_snippet
         )
         assert ssh_key in rex_snippet
+        if module_sync_kickstart_content.os.major == '9':
+            assert f'timesource --ntp-server {ntp_server_value}' in rex_snippet
```

Comment:
This looks a different scenario, not something related to rex snippets, so I feel this should go into a separate a test for ntp where we could validate both `timezone` commands and  `timesource` commands, wdyt?

Reply:
Yes. Updated.